Adding documentation for actions for switchbot_cloud (#33746)
home-assistant/home-assistant.github.io: source/_integrations/switchbot_cloud.markdown

- MeterPlus
- Outdoor Meter

> ## Services

Reviewer: ```suggestion
## Actions
```
In this release, we are renaming services to actions

### switchbot_cloud.list_devices

> This service lists all the available devices for the SwitchBot API.

Reviewer: ```suggestion
This action lists all the available devices for the SwitchBot API.
```

| Field name  | Description                                |
| ----------- | ------------------------------------------ |

> | unique_id   | Id of the device used by SwitchBot.        |

Reviewer: ```suggestion
| unique_id   | ID of the device used by SwitchBot.        |
```

| device_name | Name set in the SwitchBot application.     |
| device_type | Type of the device according to SwitchBot. |

### switchbot_cloud.send_command

> This service allows sending control command to devices as specified in [SwitchBot API documentation](https://github.com/OpenWonderLabs/SwitchBotAPI?tab=readme-ov-file#send-device-control-commands)

Reviewer: ```suggestion
This action allows sending control commands to devices as specified in [SwitchBot API documentation](https://github.com/OpenWonderLabs/SwitchBotAPI?tab=readme-ov-file#send-device-control-commands)
```

> | Service Data Attribute | Required | Description                                   |

Reviewer: ```suggestion
| Data attribute | Required | Description                                   |
```

| ---------------------- | -------- | --------------------------------------------- |

> | unique_id              | yes      | SwitchBot unique id.                          |

Reviewer: ```suggestion
| unique_id              | yes      | SwitchBot unique ID.                          |
```
